app/src/main/java/com/gruppo4/audioUtility/AudioUtilityManager.java

```diff
@@ -0,0 +1,171 @@
+package com.gruppo4.audioUtility;
+
+/**
+ * @author Francesco Bau'
+ */
+
+import android.content.Context;
+import android.media.AudioManager;
+import android.os.Build;
+import android.os.VibrationEffect;
+import android.os.Vibrator;
+
+import static android.content.Context.AUDIO_SERVICE;
+
+
+public class AudioUtilityManager {
+
+    public static final int MAX_PERCENTAGE = 100;
+    public static final int MIN_PERCENTAGE = 0;
+
+    private static final int VIBRATION_TIME = 500;
+    private static final int VIBRATION_DELAY = 500;
+    private static final int VIBRATION_REPEAT_CODE = 1;
+
+    public enum AUMStream {
+        ALARM,
+        RING,
+        MUSIC
+    }
+
+    /**
```

Comment:
Consider adding a brief explanation even if the method is self-explaining. It's good practice.

Reply:
Ok, will do it.